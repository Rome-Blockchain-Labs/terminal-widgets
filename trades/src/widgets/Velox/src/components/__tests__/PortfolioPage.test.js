import { shallow } from 'enzyme';

import { PortfolioPage } from '../PortfolioPage';

describe('PortfolioPage component render', () => {
  it('PortfolioPage snapshot added', () => {
    const component = shallow(<PortfolioPage />);
    expect(component).toMatchSnapshot();
  });
});
