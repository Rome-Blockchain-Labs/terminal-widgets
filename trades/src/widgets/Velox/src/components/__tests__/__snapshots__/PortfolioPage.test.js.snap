// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortfolioPage component render PortfolioPage snapshot added 1`] = `
<Fragment>
  <div
    className="auth__swap flex --just-center --align-center"
  >
    Here should be PortfolioPage content
  </div>
</Fragment>
`;
